theano/sandbox/cuda/opt.py

```diff
@@ -1902,7 +1895,7 @@ def local_gpu_downsample_factor_max(node):
             return
         x, = node.inputs
         if (x.owner and isinstance(x.owner.op, HostFromGpu)):
-            gpu_ds = GpuDownsampleFactorMax(node.op.ds, node.op.ignore_border)
+            gpu_ds = GpuDownsampleFactorMax(**node.op._props_dict())
```

Comment:
This line create an instance of an op. But the original CPU op have more props that then GPU one. So revert this part.